src/deploy/osp_deployer/director.py

```diff
@@ -678,10 +678,25 @@ def setup_net_envt(self):
         logger.debug("Configuring network-environment.yaml for overcloud")
 
         network_yaml = self.templates_dir + "/network-environment.yaml"
-
+        octavia_yaml = self.templates_dir + "/octavia.yaml"
+        
         self.upload_file(self.settings.network_env_yaml,
                          network_yaml)
-
+                         
+        if self.settings.octavia_user_certs_keys is True:
+          self.upload_file(self.settings.certificate_keys_path,
+                           self.templates_dir + "/cert_keys.yaml")
+          self.run_tty('sed -i "s|OctaviaGenerateCerts:.*|' + 
+            'OctaviaGenerateCerts: ' +
+            'False' + '|" ' + 
+            str(octavia_yaml))
+        
+        if self.settings.octavia_user_certs_keys is False:
+          self.run_tty('sed -i "s|OctaviaGenerateCerts:.*|' + 
+            'OctaviaGenerateCerts: ' +
+            'True' + '|" ' + 
```

Comment:
can you make is lowercase (true) to match up with default template s case

Reply:
this has been fixed